packages/vats/src/proposals/upgrade-provisionPool-proposal.js

```diff
@@ -59,35 +60,47 @@ export const upgradeProvisionPool = async (
     E(electorateCreatorFacet).getPoserInvitation(),
   ]);
 
+  const readCurrentGovernedParams = async () => {
+    await null;
+
+    const params = await E(ppPublicFacet).getGovernedParams();
+    return harden({
+      PerAccountInitialAmount: params.PerAccountInitialAmount.value,
+    });
+  };
+  const governedParamOverrides = await readCurrentGovernedParams();
```

Comment:
That seems like a lot of extra work to go through. The `await null` is unnecessary. I think this is equivalent to the following. Did I miss something?
```suggestion
  const params = await E(ppPublicFacet).getGovernedParams();
  const governedParamOverrides = harden({
      PerAccountInitialAmount: params.PerAccountInitialAmount.value,
    });;
```

Reply:
My goal was to follow the same design as you did for upgrade-vaults, but I agree that in this case it is a lot of extra work.
Thank you for noticing.
Change request implemented